spec/index.html

```diff
@@ -495,14 +522,28 @@ <h3>Serializing RDF Terms</h3>
         without syntax to denote what kind of term it is. The encoding quoting rules of CSV format must be used.</p>
       <p>Blank nodes use the <code>_:label</code> form from Turtle and SPARQL. Use of the same label indicates the
         same blank node within the result set but has no significance outside the result set.</p>
+      <p>Quoted triples are enclosed in <code>&lt;&lt; <em>subject</em> <em>predicate</em> <em>object</em> &gt;&gt;</code>,
+         where the <em>subject</em>, <em>predicate</em>, and <em>object</em> terms are recursively serialized.
+         The serializations of <em>subject</em>, <em>predicate</em>, and <em>object</em>
+         <em class="rfc2119" title="must">must</em> be separated by a single space character
+         (<small>SPACE</small>, code point 32, <code>U+0020</code>). The single space character preceding <em>subject</em>,
+         and the single space character following <em>object</em> are optional, which are mainly valuable for human readability.
+         If the <em>object</em> is a literal, then it <em class="rfc2119" title="must">must</em> be encapsulated within
+         a pair of quotation marks <code>""</code>.</p>
       <p>Fields containing any of <code>"</code> (<small>QUOTATION MARK</small>, code point 34, <code>U+0022</code> in 
         Unicode [[UNICODE]]),
         <code>,</code> (<small>COMMA</small>, code point 44, <code>U+002C</code>), <code>LF</code> (code point 10,
         <code>U+000A</code>) or <code>CR</code> (code point 13, <code>U+000D</code>) must be quoted using the quoting
         mechanism of RFC4180 [[RFC4180]].
         Fields are limited by a pair of quotation marks <code>"</code> (code point <code>U+0022</code>). Within quote strings, all
         characters except <code>"</code>, including new line characters have their exact meaning — newlines do not
-        end a CSV record. Inline <code>"</code> is written using a pair of quotation marks <code>""</code>.</p>
+        end a CSV record. Inline <code>"</code> is written using a pair of quotation marks <code>""</code>.
+        This quoting mechanism is applied recursively for terms in quoted triples.</p>
+      <p id="csv-terms-escaping" class="note">
+        Since quoted triples with literals in the object position are encapsulated within a pair of quotation marks <code>""</code>,
+        the full quoted triple will always be encapsulated within quotation marks as well,
+        which requires the subsequent escaping of the quotation marks encapsulating the literal.
+        Such cases will result in quoted triples in the form of <code>"&lt;&lt; <em>subject</em> <em>predicate</em> ""<em>object-literal</em>"" &gt;&gt;"</code>.</p>
```

Comment:
```suggestion
        Since literals in the object position of quoted triples are encapsulated within a pair of quotation marks <code>"</code>, and
        the full quoted triple is always encapsulated within quotation marks as well,
        this requires the subsequent escaping of the "inner" quotation marks encapsulating the literal.
        Such cases will result in quoted triples in the form of <code>"&lt;&lt; <em>subject</em> <em>predicate</em> ""<em>object-literal</em>"" &gt;&gt;"</code>.</p>
```